diff --git a/postinstall.js b/postinstall.js
index 78e757cbf22fd599e99307531fd1163c35da0528..3e1ada9b88e748934b69868ca242a0edae834470 100644
--- a/postinstall.js
+++ b/postinstall.js
@@ -97,6 +97,9 @@ function copyPlatformBinaries(platform) {
   /**
    * Copy the PPX
    */
+  if (fs.existsSync(path.join(__dirname, "ppx"))) {
+    fs.rmSync(path.join(__dirname, "ppx"));
+  }
   fs.copyFileSync(
     path.join(__dirname, "ppx-" + platform),
     path.join(__dirname, "ppx")
@@ -106,7 +109,9 @@ function copyPlatformBinaries(platform) {
   /**
    * Copy the Relay compiler
    */
-
+  if (fs.existsSync(path.join(__dirname, "rescript-relay-compiler.exe"))) {
+    fs.rmSync(path.join(__dirname, "rescript-relay-compiler.exe"));
+  }
   fs.copyFileSync(
     path.join(
       __dirname,
